doc/winsetup.rst

```diff
@@ -0,0 +1,129 @@
+Setting up Jackal with Windows
+==============================
+
+The Clearpath Jackal does not ship with Windows installed by default. However, the core Jackal ROS nodes 
+are enabled on Windows 10, and ROS on Windows is supported by Microsoft. This guide will cover installing 
+Windows on your Jackal, installing ROS & Jackal ROS components, updating firmware and configuring Windows 
+for use.
+
+The following instructions assume that the Jackal is connected to a keyboard and mouse during setup. Once 
+configured, these can be removed and remote access can be leveraged.
+
+For more information on ROS on Windows, please visit `Microsoft's ROS documentation`_.
+
+.. _Microsoft's ROS documentation: http://aka.ms/ros
+
+Installing Windows 10
+---------------------
+
+ROS1 is supported by Microsoft for Windows 10 Desktop and Windows 10 IoT Enterprise. Windows 10 IoT Enterprise
+is an embedded version of Windows 10 for dedicated purpose devices, such as a Robot.
+
+1. You can `download a trial`_ of Windows 10 IoT Enterprise Long Term Service (LTSC) x64 build, use a Windows license from your Enterprise agreement, or a commercially purchased version of Windows 10 Desktop.
+2. Create a `USB Boot stick`_ and copy the contents of the iso generated from the image above.
+3. Boot Windows and install following the prompts.
+4. Install ROS on Windows using the `ROS Wiki instructions`_.
+5. Please follow the instructions closely, as ROS is sensitive to the operating environment. 
+
+This will install the development platform on the Jackal, which allows you to develop your robotics solutions. Once you've 
+finalized your development, you can build a deployment package and deploy your solutions using Chocolatey. 
+
+If you see any problems with Jackal, please `get in touch`_ and we'll see if we can get you sorted out.
+
+If you have Windows specific questions or problems, Microsoft offers `community support through Github`_ and `paid support`_. 
+
+.. _download a trial: https://www.microsoft.com/en-us/evalcenter/evaluate-windows-10-enterprise
+.. _ROS Wiki instructions: https://wiki.ros.org/Installation/Windows
+.. _USB Boot Stick: https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/install-windows-from-a-usb-flash-drive
+.. _get in touch: https://support.clearpathrobotics.com/hc/en-us/requests/new
+.. _community support through Github: https://github.com/ms-iot/rosonwindows
+.. _paid support: http://aka.ms/ros/support
+
+Installing Jackal software
+--------------------------
+
+Once you've installed Windows and ROS, It's time to install the Jackal software. 
+
+.. code-block:: batch
+
+    mkdir c:\ws\src
+    cd c:\ws\src
+    git clone https://github.com/jackal/jackal_robot
+    git clone https://github.com/jackal/jackal
+    git clone https://github.com/jackal/jackal_firmware_ms
+    catkin_make
```

Comment:
We're missing a cd .. command here; running catkin_make directly in the src directory fails.